recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37]
```

Comment:
What does this do?

Reply:
The line tells conda to only test the package for Python versions 3.7 and above.

Evidently conda forge tests the package against a range of Python versions.  Currently it tests against Python  3.6, 3.7, 3.8 and 3.9.  Presumably a few months ago it also included 3.5 and in a few months from now it will drop 3.6 . 

Our setup.py has a requirement of Python>= 3.7, which I believe is to overcome a cartopy bug.  conda forge tests against Py 3.6 failed because it didn't look into setup.py and find our Py 3.7 requirement.

Does that make sense?

Reply:
Oh, nice! That definitely makes sense. xarray has also dropped py3.6, so we shouldn't test it because of that too. So yeah, nicely done.